src/styles/_mixins.scss

```diff
@@ -6,4 +6,14 @@
 
   @mixin expand-transition {
     transition: width 0.3s ease-in-out;
+  }
+
+  @mixin chip {
```

Comment:
I suggest to create a simple component e.g. `BaseChip` instead of applying the same styles to elements. It may reduce the app size at the end and may be easier to manage :)

I saw you are already familiar with slots (you used it in `BaseButton`) - it may be another use case.